reference-adapter/ifix-department-entity-service/src/main/resources/db/migration/main/V202211131027__department_entity_ddl.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE IF NOT EXISTS public.department_entity
+(
+    id character varying(255) COLLATE pg_catalog."default" NOT NULL,
+    children character varying(5000) COLLATE pg_catalog."default",
```

Comment:
Children will not be a comma-separated string. Make a separate table to store the parent-child relationship. 
The same applies to the hierarchy level too.

Reply:
It requires discussion. It's stable with this version. Changing into table can lead structural change and lot of places changes would be require.
Please consider time.